deploy/internal/statefulset-core.yaml

```diff
@@ -141,3 +141,23 @@ spec:
               valueFrom:
                 resourceFieldRef:
                   resource: limits.memory
+        - name: noobaa-log-processor
+          image: noobaa-core:s3-tests
+          command: ["/root/node_modules/noobaa-core/src/deploy/NVA_build/noobaa_logs.sh"]
+          volumeMounts:
+            - name: logs
+              mountPath: /log
+          resources:
+            requests:
+              cpu: "900m"
```

Comment:
let's use less cpu\mem for now. maybe 200m CPU and 500MB.

Reply:
done